readme.txt

```diff
@@ -4,7 +4,7 @@ Contributors: ModernTribe, borkweb, zbtirrell, barry.hughes, bordoni, brianjesse
 Tags: events, calendar, event, venue, organizer, dates, date, google maps, conference, workshop, concert, meeting, seminar, summit, class, modern tribe, tribe, widget
 Donate link: http://m.tri.be/29
 Requires at least: 3.9
-Tested up to: 4.7.2
+Tested up to: 4.7
```

Comment:
Should have caught this yesterday - also, it's easy to fix when we next deploy from this branch - but it looks like we retained the wrong version number here